### DISCUSSION TITLE
Installing VDH OK but Companion not installing need terminal sudo install

### DISCUSSION THREAD

**Discussion by needhelpjal** [aclap-dev/video-downloadhelper#37] [Help and Q&A]
### Your question

Hi I use ubuntu studio 18.4 - Firefox 113.0.2 (64-bit) - video download helper wants me to upgrade - Companion application upgrade required⊗ - You are running the companion application version 1.3.0 but this feature requires version 2.0.9 - But how do i do the up grade in a terminal - Because Using the install manager, the install fails to install, the install manager has never worked with installing any app - I can only install apps via the terminal - Can any one help

**Comment by paulrouget** [comment]
In a terminal, do this:

```
sudo apt install -y wget flatpak
wget https://github.com/aclap-dev/vdhcoapp/releases/latest/download/vdhcoapp-linux-x86_64.deb
sudo dpkg -i ./vdhcoapp-linux-x86_64.deb
flatpak permission-set webextensions net.downloadhelper.coapp snap.firefox yes
```
